content/kubermatic/master/installation/start_kkp/guides/setup_repository/_index.en.md

```diff
@@ -150,9 +167,9 @@ In addition to above, configure following secrets for selected cloud provider.
 | `ARM_CLIENT_SECRET`     | The Azure Client Secret for client authentication                    |
 {{% /tab %}}
 {{% tab name="GCP" %}}
-| Secret Name             | Description                                                                                |
-| ------------------------| -------------------------------------------------------------------------------------------|
-| `GOOGLE_CREDENTIALS`    | The service account key (content of `${SERVICE_ACCOUNT_NAME}-sa-key.json`) from above step |
+| Secret Name             | Description                                                                                                                                                                                        |
+| ------------------------| ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `GOOGLE_CREDENTIALS`    | The service account key (content of `${SERVICE_ACCOUNT_NAME}-sa-key.json`) from above step. For **Bitbucket**, base64 encoded value is required (`base64 -w0 ${SERVICE_ACCOUNT_NAME}-sa-key.json`) |
```

Comment:
didn't you have another variable used for this purpose on Bitbucket? and `GOOGLE_CREDENTIALS` is then set from that env.variable?

Reply:
Do you think that's ok?

<img width="1260" alt="image" src="https://user-images.githubusercontent.com/4434544/158574190-07512953-d567-432f-9d48-c8bedec9f477.png">